bin/loadmeta.jl

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/env julia
 
 using Base: thispatch, thisminor, nextpatch, nextminor
-using Base.Pkg.Reqs: Reqs, Requirement
-using Base.Pkg.Types
 using Base.LinAlg: checksquare
 using Base.Random: UUID
 using SHA
 
-## Computing UUID5 values from (namespace, key) pairs ##
+import Pkg3.Pkg2
```

Comment:
😄

Reply:
In `Pkg4` we will have `Pkg4.Pkg3.Pkg2`.

Reply:
The Russian doll of package managers.